doc/drop_counters/drop_counters_HLD.md

```diff
@@ -100,12 +106,157 @@ Users must be able to use all counters and drop reasons provided by the underlyi
+--------  --------  --------------  -------------------  --------------
+DEBUG_0   RX_LEGIT  PORT_INGRESS    SMAC_EQUALS_DMAC     Legitimate port-level RX pipeline drops
+                                    INGRESS_VLAN_FILTER
+DEBUG_1   TX_LEGIT  PORT_EGRESS     EGRESS_VLAN_FILTER   Legitimate port-level TX pipeline drops
+DEBUG_2   RX_LEGIT  SWITCH_INGRESS  TTL                  Legitimate switch-level RX pipeline drops
+```
+
+### 3.1.3 Displaying the current counts
+
+```
+$ show drops
+          IFACE    STATE      RX_ERR     RX_DRP    RX_DISC    RX_LEGIT    TX_ERR    TX_DRP    TX_DISC    TX_LEGIT
+---------------  -------  ----------   --------  ---------  ----------  --------  --------  ---------  ----------
+      Ethernet0        U           0          0       1500        1500         0         0          0           0
+      Ethernet4        U           0          0        300         250         0         0          0           0
+      Ethernet8        U           0          0          0           0         0         0          0           0
+     Ethernet12        U           0          0       1200         400         0         0          0           0
+
+         DEVICE    STATE    RX_LEGIT  
+---------------  -------  ----------  
+ABCDEFG-123-XYZ        U        2000
+
+$ show drops --type=PORT
+     IFACE    STATE    RX_ERR    RX_DRP    RX_DISC    RX_LEGIT    TX_ERR    TX_DRP    TX_DISC    TX_LEGIT
+----------  -------  --------  --------  ---------  ----------  --------  --------  ---------  ----------
+ Ethernet0        U        0         0       1500        1500         0         0          0           0
+ Ethernet4        U        0         0        300         250         0         0          0           0
+ Ethernet8        U        0         0          0           0         0         0          0           0
+Ethernet12        U        0         0       1200         400         0         0          0           0
+
+$ show drops --contains "LEGIT"
+          IFACE    STATE    RX_LEGIT    TX_LEGIT
+---------------  -------  ----------  ---------- 
+      Ethernet0        U           0           0       
+      Ethernet4        U           0           0       
+      Ethernet8        U           0           0          
+     Ethernet12        U           0           0 
+
+         DEVICE    STATE    RX_LEGIT
+---------------  -------  ----------  
+ABCDEFG-123-XYZ        U        2000
+```
+
+### 3.1.4 Clearing the counts
+```
+$ sonic-clear drops
+```
+
+### 3.1.5 Configuring counters from the CLI
+```
+$ config drops init --counter="DEBUG_3" --alias="EXAMPLE" --type="SWITCH_EGRESS" --desc="example" --reasons=["L2_ANY", "L3_ANY"]
+Initializing DEBUG_3 as TX_LEGIT...
+
+Counter   Alias     Type           Reasons  Description
+-------   --------  -------------  -------  -----------
+DEBUG_3   TX_LEGIT  SWITCH_EGRESS  L2_ANY   Legitimate switch-level TX pipeline drops
```

Comment:
These values don't match the given CLI example (e.g. alias, desc) - confusing.

Reply:
Thanks for catching this! It's fixed now. :)